### DISCUSSION TITLE
`cus-cn.tex` 关于 `\V` 和 `\verb` 的问题

### DISCUSSION THREAD

**Discussion by xkwxdyy** [Sophanatprime/cus#35] [Q&A] [answered]
最近在学习 `cus-cn.tex` 的代码，来学习一下一些写手册的代码。

## 关于 `\V` 和 `\verb`
手册里的 `syntax` 环境中用到了这两个
https://github.com/Sophanatprime/cus/blob/aa23963ee748c79f5a481fbf491532c3dea03fbd/cus-cn.tex#L421-L424
https://github.com/Sophanatprime/cus/blob/aa23963ee748c79f5a481fbf491532c3dea03fbd/cus-cn.tex#L442-L446

但我不太明白，为什么一个用 `\verb` 一个用 `\V`呢？

**Comment by Sophanatprime** [comment, marked as answer]
某个版本之前没有 `\V`，所以就用 `\verb` 了，没改正过来。现在应当使用 `\V` 和 `\V*`。包括 `\marg` 和 `\meta` 的平替 `<{..}>` `<>`，目前也是几种写法并存的。

**Reply by xkwxdyy** [reply]
也就是优先用 `\V`, `\V*`, 和 `<{..}> <>` 是吗？
请问 `\V` 和 `\V*` 有什么区别呢？

**Reply by Sophanatprime** [reply]
`syntax` 环境中有 `\let\V\Verbatimize`。不带星号的和 `\fakeverb` 类似，带星号的和 `\verb` 类似，都不切换字体，文档第 8 页有说明。

**Reply by Sophanatprime** [reply]
`<{..}>` `<>` 只是语法糖，作用和 `\marg` `\meta` 完全一样，`&&` `&#` 这些也是，文档有说明。

**Reply by xkwxdyy** [reply]
感谢
